app/data/1_0001.json

```diff
@@ -4859,7 +4890,12 @@
                                             "mandatory": false,
                                             "options": [],
                                             "q_code": "2620",
-                                            "type": "Percentage"
+                                            "type": "Percentage",
+                                            "validation": {
+                                                "messages" : {
+                                                    "INTEGER_TOO_LARGE": "The maximum value allowed is 100. Please correct your answer."
```

Comment:
@benjefferies I tested moving the validation message overrides to the Question level, rather than having them on each percentage answer. But the messages didn't get resolved correctly - it just displayed the global messages. Perhaps its worth creating a new card to investigate if specifying validation overrides on the Question makes sense and to fix that. But in terms of this particular card, I think this is the least amount of duplication I am able to achieve.